CI/physmon/workflows/physmon_trackfinding_4muon_50vertices.py

```diff
@@ -69,6 +69,11 @@
         setup.trackingGeometry,
         setup.field,
         rnd=rnd,
+        postSelectParticles=ParticleSelectorConfig(
+            pt=(0.9 * u.GeV, None),
```

Comment:
I think, I didn't fully understand the system, when we have `0.5 GeV` and when `0.9 GeV`. In this case we changed it, in some other place we kept.

Reply:
It doesn't matter all too much. I raised it for the single particle cases since we produce >= 1GeV anyways. For ttbar I ended up not doing it because we actually reconstruct down to 0.5 GeV for now and the plots change significantly.